README.md

```diff
@@ -92,6 +92,22 @@ Other operating systems, such as Microsoft Windows, which access files in a case
 
 Best practice is to use `data-e2e="your_test_id"` element attributes to select specific elements, eg `cy.get('[data-e2e="cta-button-1"]').click()` instead of `cy.get('.container .infobox a.button').contains('DOWNLOAD').click()`.
 
+|                       | Included in Cypress Test Production?     |  What is it used for?                                                                |
+|-----------------------|-|-------------------------------------------------------------------------------------------------------------------------------|
+| check_links.js        |✘ (takes a long time to finish running)   | Detect broken links throughout the website. Result is saved in `logs` folder         |
+| app_to_web.js         |✓                                         | Detect broken links throughout the website coming from cwa-webserver                |
+| applink_spec.js       |✓                                         | Detect broken links or missing OS badges to download mobile app in home page         |
+| blog_spec.js          |✓                                         | Check if blog archive is accessible                                                   |
+| check_anchor_links.js |✓                                         | Detect broken anchor links throughout the website. Result is saved in `logs` folder  |
+| check_videos.js       |✓                                         | Detect broken videos throughout the website                                          |
+| eventRegistration.js  |✓                                         | Checks that all fields in event-registration exist and fill them to create a QR code |
+| faq.js                |✓                                         | Simulate a FAQ search and the result for mobile and desktop view                     |
+| hotline_spec.js       |✓                                         | Verify that Hotline APP and TAN data is correcly displayed at home page              |
+| mime.js               |✓                                         | Check that CSS and JS files have the correct myme                                    |
+
+To run all test included in Cypress Test Production execute:
+`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headed` => test results are printed in console, also you can see browser's movements
+`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in console, you can`t see browser's movements
```

Comment:
```suggestion
`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in console, you can't see browser's movements
```
I'd suggest to replace ` with ' here for consistency reasons. In markdown the text afterwards is also highlighted in blue because it is considered as the start of a code section, like seen before in the same line